CRM/Activityreport/Data.php

```diff
@@ -151,6 +151,17 @@ protected static function populateMultiValuesRow(array $row, array $fields, $off
     $found = true;
     $i = 0;
 
+    /*
+     * Check how many combinations of one $row needs to be created.
```

Comment:
you can separate this into new private method : 

`
$info['multiValuesTotal'] = $this->getTotalCombinations($fields)
`